tests/serde-se.rs

```diff
@@ -189,7 +192,7 @@ mod without_root {
                         e
                     ),
                 }
-                assert_eq!(buffer, "");
+                assert!(buffer.is_empty());
```

Comment:
`assert_eq` used for purpose because it is from `pretty_assertions` crate and will produce a diff in case of failure